Add :nosort: & :ownline: display tags to the readme (#223)
Kungsgeten/org-brain: README.org

     entries.
- =:nosiblings:= :: You may have an entry with lots of children, and when you visualize one of these children you might not want to see the siblings from this parent. A good example would be if you have an =index= entry or similar. By tagging the parent with =nosiblings= the parent's children will not show siblings from that parent. You can change the tag name by modifying =org-brain-exclude-siblings-tag=.

The following tags modify the way how information is shown when an
entry is visualized.

- =:ownline:= :: Make each child of the tagged entry appear on its own

>                  line when the tagged entry is visualized. This does

Reviewer: ```suggestion
                 line when the tagged entry is visualized. This
```

>                  only affect the tagged entry. I works akin to

Reviewer: ```suggestion
                 only affects the tagged entry. It works akin to
```

                 temporarily setting =org-brain-child-fill-column-sexp=
                 to =0=.

- =:nosort:= :: Display each child of the tagged node in the order the

>                 children are listed in the file. This affects the

Reviewer: ```suggestion
                children are listed in the file, rather than in the sorted order determined by =org-brain-visualize-sort-function=. This affects the
```

>                 children are listed in the file, rather than in the sorted order determined by =org-brain-visualize-sort-function=. This affects the

Reviewer: You might want to re-break the lines. I wasn't able to do this with GitHub's "suggestions" feature...

Author: Done
